aasdk_proto/AVChannelSetupStatusEnum.proto

```diff
@@ -1,31 +1,31 @@
 /*
-*  This file is part of aasdk library project.
-*  Copyright (C) 2018 f1x.studio (Michal Szwaj)
-*
-*  aasdk is free software: you can redistribute it and/or modify
-*  it under the terms of the GNU General Public License as published by
-*  the Free Software Foundation; either version 3 of the License, or
-*  (at your option) any later version.
-
-*  aasdk is distributed in the hope that it will be useful,
-*  but WITHOUT ANY WARRANTY; without even the implied warranty of
-*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-*  GNU General Public License for more details.
-*
-*  You should have received a copy of the GNU General Public License
-*  along with aasdk. If not, see <http://www.gnu.org/licenses/>.
-*/
-
-syntax="proto2";
-
-package f1x.aasdk.proto.enums;
-
-message AVChannelSetupStatus
-{
-    enum Enum
-    {
-        NONE = 0;
-		FAIL = 1;
-		OK = 2;
-    }
-}
+*  This file is part of aasdk library project.
+*  Copyright (C) 2018 f1x.studio (Michal Szwaj)
+*
+*  aasdk is free software: you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation; either version 3 of the License, or
+*  (at your option) any later version.
+
+*  aasdk is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with aasdk. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+syntax="proto2";
+
+package f1x.aasdk.proto.enums;
+
+message AVChannelSetupStatus
+{
+    enum Enum
+    {
+        NONE = 0;
+		FAIL = 1;
```

Comment:
tabs vs spaces on these?

Reply:
tabs versuss spaces across _all_ these files are untouched -- all I did was fix line endings to be native.